src/components/sidebar/components/sidebar-horizontal-rule/index.tsx

```diff
@@ -0,0 +1,7 @@
+import s from './sidebar-horizontal-rule.module.css'
+
+const SidebarHorizontalRule = () => {
```

Comment:
Just thinking outloud, but I wonder if it's needed to have the explicit `SidebarComponentName` pattern with these. It may read easier in the sidebar component and working with it to have it be like: 

```
<Sidebar >
  <SectionHeading>Yo</SectionHeading>
  <HorizontalRule />
</Sidebar>
```

Given that these components will always be used within the context of the sidebar, as children or a composed component it seems. I could see it either way though! It's up to you.

Reply:
So the reason I prefixed these with `Sidebar` has to do with codebase-wide searching. `SectionHeading` and `HorizontalRule` are both generic enough that we could eventually create those components again later for other parts of the app. That would leave us with two (or more, potentially) versions of components with the same name, which makes searching for usage of a component harder.

Reply:
Will +1 to both 😝 I can see this either way, and personally lean towards the `Sidebar` prefix for extra clarity, and for code-search & future-maybe-duplicate-name reasons.

Reply:
The search-ability aspect and potential name duplicates makes sense to me!